packages/services/f-analytics/src/services/analytics.mapper.js

```diff
@@ -103,13 +104,14 @@ const mapUserAgent = req => {
 export const mapServerSidePlatformData = ({ platformData, req } = {}) => {
     const userPercent = getCookie('je-user_percentage', req);
 
+    /* eslint-disable dot-notation */
     const mappedData = {
         ...platformData,
         jeUserPercentage: userPercent || platformData.jeUserPercentage,
-        environment: process.env.justEatEnvironment || platformData.environment,
-        version: process.env.FEATURE_VERSION || platformData.version,
-        instancePosition: process.env.INSTANCE_POSITION || platformData.instancePosition,
-        isPilot: process.env.IS_PILOT || platformData.isPilot
+        environment: process.env['justEatEnvironment'] || platformData.environment,
```

Comment:
What's the error out of interest?

Reply:
No error just doesn't evaluate but 'bracket notation' works ... read up a bit and it seems 'bundling' can cause issues with 'dot notation'.